appengine/endpoints/app.rb

```diff
@@ -56,6 +66,19 @@ def auth_info
   auth_info
 end
 
+# Auth info with Firebase Auth.
+get "/auth/info/firebase" do
+  auth_info
+end
+options "/auth/info/firebase" do
```

Comment:
nit: can you add a newline above `options "..." do` so the handlers are separated by 1 newline?